GiEnJul/Controllers/AdminController.cs

```diff
@@ -84,6 +85,26 @@ public async Task<FileStreamResult> DownloadExcelDeliveryLocationAsync(string lo
             using var wb = ExcelGenerator.Generate(_mapper.Map<IEnumerable<DeliveryExcel>>(connections));
             return wb.Deliver("leveranse_liste.xlsx");
         }
+
+        [HttpGet("connections/{location}")]
+        [Authorize(Policy = "ReadConnection")]
+        public async Task<IList<GetConnectionDto>> GetConnectionsByLocationAsync(string location)
+        {
+            var eventName = await _eventRepository.GetActiveEventForLocationAsync(location);
+            var completed = await _connectionRepository.GetAllConnectionsByLocation(eventName, location);
+            var Dto = _mapper.Map<List<GetConnectionDto>>(completed);
```

Comment:
Since this is a list the variable name could be connecitonDtos or connections or getConntectionDtos or something

Reply:
should be fixed with b6d17bd